llama_stack/apis/memory_banks/client.py

```diff
@@ -94,7 +100,7 @@ async def run_main(host: str, port: int, stream: bool):
 
     # register memory bank for the first time
     response = await client.register_memory_bank(
-        VectorMemoryBankDef(
+        VectorMemoryBank(
```

Comment:
hm, this is confusing because there was one place above (where I commented) where the signature of the `register_memory_bank()` method wasn't quite like this.

Reply:
ah this is not updated. fixed now.